schema/filespec-schema.json

```diff
@@ -83,6 +83,12 @@
         "description": "If true, archive file is extracted after the operation. The archived file itself is deleted. The supported archive types are: zip, tar; tar.gz; and tgz.",
         "default": "false"
       },
+      "bypass-archive-validation": {
+        "type": "string",
+        "enum": ["true", "false"],
+        "description": "If true, bypass archive validation that usually happens before exploding.",
```

Comment:
If true, bypass archive validation that usually happens before exploding.
-->
If true, bypass the security inspection the archive go through before it is unarchived.